moveit_ros/planning/trajectory_execution_manager/src/trajectory_execution_manager.cpp

```diff
@@ -1430,6 +1437,61 @@ bool TrajectoryExecutionManager::executePart(std::size_t part_index)
   }
 }
 
+bool TrajectoryExecutionManager::waitForRobotToStop(const TrajectoryExecutionContext &context, double wait_time)
+{
+  if (allowed_start_tolerance_ == 0)  // skip validation on this magic number
```

Comment:
``allowed_start_tolerance_`` is a double, which as I understand it you should never compare directly to ``0``